docker/mitre-docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+services:
+  ecrnow-ui:
+    image: ecrnow-ui
+    container_name: ecrnow-ui
+    restart: on-failure
+    ports:
+      - "3001:3000"
```

Comment:
I'm assuming we're mapping 3001 -> 3000 because we're already using 3000 for our own backend service app? Would you mind adding that as a comment?